rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -209,6 +213,14 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 return true;
             }
         });
+
+        String lastSyncTimestampStr = RDTApplication.getInstance().getContext().allSharedPreferences().getPreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP);
```

Comment:
`RDTApplication.getInstance().getContext().allSharedPreferences()` is also used elsewhere in this method. Could you define it once in a variable?

Reply:
Also, could we rename all instances of `lastSyncTimestamp` to `latestSyncTimestamp`.

Reply:
done